### PR TITLE
Feat paddy modstubreadmetransfer

### LINKED ISSUE #88: [DOC]Remove unnecessary documentation from mod stub repo readme.md
Doc's such as debug information, unity setup etc are better directed toward the modding wiki than duplicated here. If we have any JVL specific implementations such as J/VLmock_ prefixing for asset creation then include these, but only link to generic information which is tangential to our project.

### PR DESCRIPTION
**Pull request by paddywaan**
Implemented various tutorials contributing to #38, resolved #88. xref's in documentation are currently broken, seeking workflow advice.

### CODE REVIEW

File: JotunnLib/Documentation/tutorials/utils/config.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿# Persistent & Synced Configurations
+
+Jötunn itself does not provide any implementations or abstractions for persisent configurations. We do however respect [BepInEx.ConfigEntry](https://bepinex.github.io/bepinex_docs/master/articles/dev_guide/plugin_tutorial/3_configuration.html)'s, their various properties, as well as their [ConfigurationManager](https://github.com/BepInEx/BepInEx.ConfigurationManager) properties. Furthermore we have implemented a method of enforcing server side sync on specific configs via the [ConfigurationManagerAttributes](xref:ConfigurationManagerAttributes) `IsAdminOnly` flag.
```
Comment (Algorithman) [inline JotunnLib/Documentation/tutorials/utils/config.md:3]: For the ConfigurationManagerAttributes we should add a hint, so mod developers will use JVL's implementation of the ConfigurationManagerAttributes. The original implementation does not include the admin only flag. (so xref to JotunnLib.ConfigurationManagerAttributes)

File: JotunnLib/Documentation/tutorials/data/unity.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- One unity project that will be made entirely by [uTinyRipper](https://sourceforge.net/projects/utinyripper/files/latest/download) (explained below in the `Making our Item Prefab` chapter)
+- One [unity stub project](https://github.com/Valheim-Modding/JotunnModStub/tree/master/JotunnModUnity) that we will be at first entirely empty, that is made like so
+
+### Unity Stub
+1. [Download](https://public-cdn.cloud.unity3d.com/hub/prod/UnityHubSetup.exe) UnityHub directly from Unity or install it with the Visual Studio Installer via `Individual Components` -> `Visual Studio Tools for Unity`.
+2. You will need an Unity account to register your PC and get a free licence. Create the account, login with it in Unity Hub and get your licence via `Settings` -> `Licence Management`.
+3. Install Unity Editor version 2019.4.20f. [download that and install it](https://unity3d.com/unity/whats-new/2019.4.20).
+4. Open Unity and add the ValheimModUnity project.
+5. Install the `AssetBundle Browser` package in the Unity Editor via `Window`-> `Package Manager`.
```
Comment (sirskunkalot) [inline JotunnLib/Documentation/tutorials/data/unity.md:58]: thats doubled with 8.

File: JotunnLib/Documentation/tutorials/data/unity.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+6. Copy all `assembly_*.dll` from `<ValheimDir>\valheim_Data\Managed` into `<ValheimModStub>\ValheimModUnity\Assets\Assemblies`.
+
+ **Copy the assemblies to the new project directly via the filesystem - don't import the dlls via Unity**.
+
+![Game Assemblies](https://i.imgur.com/yVOLfOa.png) ![Assemblies Folder](https://i.imgur.com/4Zun1yU.png)
+
+7. Go back to the Unity Editor and press `Ctrl+R`. This should reload all files and add the Valheim assemblies under `Assets\Assemblies`
+8. Click on `Window` -> `PackageManager` and search for `AssetBundleBrowser` and install it.
```
Comment (sirskunkalot) [inline JotunnLib/Documentation/tutorials/data/unity.md]: thats doubled with 5.

File: JotunnLib/Documentation/tutorials/utils/config.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    Config.Bind("JotunnLibTest", "FloatValue1", 750f, new ConfigDescription("Server side float", new AcceptableValueRange<float>(0f,1000f), new ConfigurationManagerAttributes { IsAdminOnly = true }));
+    Config.Bind("JotunnLibTest", "IntegerValue1", 200, new ConfigDescription("Server side integer", null, new ConfigurationManagerAttributes { IsAdminOnly = true }));
+    Config.Bind("JotunnLibTest", "BoolValue1", false, new ConfigDescription("Server side bool", null, new ConfigurationManagerAttributes { IsAdminOnly = true }));
+    Config.Bind("JotunnLibTest", "KeycodeValue", KeyCode.F10,
+        new ConfigDescription("Server side Keycode", null, new ConfigurationManagerAttributes() { IsAdminOnly = true }));
+}
+```
+
+Here we have implemented some BepInEx configuration attributes to act as a showcase for what BepInEx has to offer, as well as our own implementation of synced attributes. This allows admins defined in the servers adminlist.txt to change the values on the fly, however clients without admin have no control over this config.
```
Comment (sirskunkalot) [inline JotunnLib/Documentation/tutorials/utils/config.md:28]: i would also mention that theses server side values are push synced to all clients connecting to the server.

**Review by Algorithman — APPROVED** [review]

**Review by sirskunkalot — APPROVED** [review]

**Pull request merged**